azurelinuxagent/common/osutil/mariner.py

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright 2018 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Requires Python 2.6+ and Openssl 1.0+
+#
+
+from azurelinuxagent.common.osutil.default import DefaultOSUtil
+
+class MarinerOSUtil(DefaultOSUtil):
+    def __init__(self):
+        super(MarinerOSUtil, self).__init__()
+        self.jit_enabled = True
+
+    def is_dhcp_enabled(self):
+        return True
+
+    def start_network(self) :
```

Comment:
NIT: Unwanted space before `:`

Change to - 
`def start_network(self):`

Reply:
Good catch. Fixed it.